### ISSUE TITLE
Switch users are unable to expand/collapse the stack view 

### ISSUE THREAD

**Issue by clane** [vmware/clarity#4647]
[Clarity Version]
3.1.1

[Test URL]
http://chrislane.info/clarity3demo1/

[WCAG 2.1 Guideline]
2.1.1 Keyboard
(Level A)
https://www.w3.org/TR/WCAG21/#keyboard

[OS]
Windows 10

[User Agents/Assistive Technologies]
Firefox 75.0, Switch, MacOs/Firefox

[Component]
Stack View

[Description]
Switch users are unable to expand/collapse the stack view because it is not accessible via keyboard.

*Note from chlane - It looks like the following element is receiving focus when using the Tab key in MacOs/Firefox. It is not normal for static elements to receive focus without the tabindex attribute in place but it is happening, maybe due to event bubbling?. 

&lt;clr-stack-block _ngcontent-ufc-c224=&quot;&quot; _nghost-ufc-c141=&quot;&quot; ng-reflect-aria-level=&quot;1&quot; ng-reflect-aria-setsize=&quot;3&quot; ng-reflect-aria-posinset=&quot;1&quot; class=&quot;stack-block stack-block-expandable ng-star-inserted&quot;&gt;

[User Impact]
The user cannot expand the stack view via keyboard.	Switch users are unable to expand/collapse the stack view because it is not accessible via keyboard.
*Note from chlane - keyboard-only users are able to expand/collapse the stack view but they will experience and extra Tab stop when the clr-stack-block element gains focus as they use the Tab key.

[Steps to Reproduce]
See the following instructions for switch usage then follow the steps below.

Windows (see Scan through keys)
https://support.microsoft.com/en-us/help/10762/windows-use-on-screen-keyboard

Mac
https://support.apple.com/en-us/HT202865

1. Using a switch, navigate to Label 2 in the Angular stack view heading with the Tab key. 
2. Try activating Label 2 with the Enter or Space Bar key. Notice that Label 2 does not expand. 

[Recommendation]
While keyup events are present within the code, and these events fire as expected in Chrome, in Firefox the keyup events are not triggering appropriately. The easiest solution would be to implement native HTML button elements. Native elements should fire click events as expected on keyboard events in all major browsers. Additionally, using native HTML will remove the need to apply role and tabindex attributes to mimic the existing native semantics.	&lt;button _ngcontent-fyp-c141=&quot;&quot; class=&quot;stack-block-label&quot; id=&quot;clr-id-11&quot; aria-expanded=&quot;false&quot; aria-posinset=&quot;2&quot; aria-level=&quot;1&quot; aria-setsize=&quot;3&quot;&gt;...&lt;/button&gt;

[Reporter Email] 
daman@damanw.com
